src/modals/proposalSelector.jsx

```diff
@@ -65,7 +65,7 @@ const ProposalSelector = () => {
       <ModalOverlay bgColor={rgba(theme.colors.background[500], 0.8)} />
       <ModalContent
         rounded='lg'
-        bg='blackAlpha.600'
+        bg='#0b0b0b'
```

Comment:
It might be good to store this color in a constants value or use css variables if possible

Reply:
Agreed. I'm going to put it in the theme.js as a const for now.